spec/integration/http.spec.js

```diff
@@ -90,13 +105,28 @@ describe( 'HTTP', function() {
 
 		harness.addResource( {
 			name: 'test',
+			middleware: [
+				function( envelope, next ) {
+					if( envelope.headers.accept ) {
+						var accept = envelope.headers.accept;
+						var match = /[.]v([0-9]*)/.exec( accept );
```

Comment:
@digitalBush actually, I take it back, most examples I've seen don't put the version specifier at the end like that. I'm sticking w/ what's there as it will match more cases (including yours which is atypical).